src/blenderbim/blenderbim/bim/__init__.py

```diff
@@ -66,6 +66,7 @@
     "covetool": None,
     "augin": None,
     "debug": None,
+    "gis": None,
```

Comment:
Perhaps put this above covetool as order determines which UI panel is shown first.